File: 01 Hands-on Activity 1 - ARG/DisplayClassmates.java
public class DisplayClassmates {

    public static void main(String[] args) {
      displayClassmates();
    }
  
    public static void displayClassmates() {
      Scanner scanner = new Scanner(System.in);
  
      System.out.print("Enter the name of your first classmate: ");
      String firstName = scanner.nextLine();
  
      System.out.print("Enter the name of your second classmate: ");
      String secondName = scanner.nextLine();
  
      System.out.print("Enter the name of your third classmate: ");
      String thirdName = scanner.nextLine();
  
      System.out.println("Your classmates are: " + firstName + ", " + secondName + ", and " + thirdName + ".");
    }
  }

File: 01 Hands-on Activity 1 - ARG/display_classmates.py
def display_classmates():
  """Gets names of three classmates from the user and displays them."""
  # Get the first classmate's name
  first_name = input("Enter the name of your first classmate: ")

  # Get the second classmate's name
  second_name = input("Enter the name of your second classmate: ")

  # Get the third classmate's name
  third_name = input("Enter the name of your third classmate: ")

  # Display the names
  print(f"Your classmates are: {first_name}, {second_name}, and {third_name}.")

# Call the function to display names
display_classmates()